File: src/manager/src/main/java/ch/unisg/edpo/manager/controllers/HealthCheckController.java
package ch.unisg.edpo.manager.controllers;

import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
class HealthCheckController {
    @GetMapping("/healthz")
    public ResponseEntity<Void> healthCheck() {
        return ResponseEntity.ok().build();
    }
}

File: src/colors/src/main/kotlin/ch/unisg/edpo/colors/controllers/HealthCheckController.kt
package ch.unisg.edpo.colors.controllers

import org.springframework.http.ResponseEntity
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class HealthCheckController {
    @GetMapping("/healthz")
    fun healthCheck() = ResponseEntity.ok().build<Unit>()
}
